neo/Wallets/VerificationContract.py

```diff
@@ -0,0 +1,114 @@
+from neo.SmartContract.Contract import Contract
+from neo.Cryptography.Crypto import Crypto
+from neo.SmartContract.ContractParameterType import ContractParameterType
+from neo.VM.OpCode import CHECKSIG
+from neo.IO.Helper import Helper as IOHelper
+
+
+class VerificationContract(Contract):
+    def __init__(self, redeem_script=None, param_list=None, pubkey_hash=None):
+        super(VerificationContract, self).__init__(
+            redeem_script=redeem_script,
+            param_list=param_list,
+            pubkey_hash=pubkey_hash
+        )
+
+    @property
+    def IsStandard(self):
+        """
+        XXXUnknown.
+
+        Returns:
+             bool: True if standard, False otherwise.
+        """
+        scp = self.Script
+
+        if len(scp) != 35:
+            return False
+
+        if scp[0] != 33 or scp[34] != int.from_bytes(CHECKSIG, 'little'):
+            return False
+
+        return True
+
+    @property
+    def ScriptHash(self):
```

Comment:
note I'm overriding this method from the `VerificationCode`  class because `self.Script` should not be a hex string. I'm being strict on data types to prevent mismatches in the future (which i've already encountered quite some).